src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,57 +1,151 @@
-import React from 'react';
+import cn from 'classnames';
+import React, { memo, useState } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
-  return (
-    <>
-      <form className="find-movie">
-        <div className="field">
-          <label className="label" htmlFor="movie-title">
-            Movie title
-          </label>
-
-          <div className="control">
-            <input
-              data-cy="titleField"
-              type="text"
-              id="movie-title"
-              placeholder="Enter a title to search"
-              className="input is-dander"
-            />
+type Props = {
+  onAddMovie: (movieId: Movie) => void;
+  errorAddMovie: boolean;
+  setErrorAddExist: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const defaultPicture = (
+  'https://via.placeholder.com/360x270.png?text=no%20preview'
+);
+
+export const FindMovie: React.FC<Props> = memo(
+  ({ onAddMovie, errorAddMovie, setErrorAddExist }) => {
+    const [movie, setMovie] = useState<Movie | null>(null);
+    const [inputValue, setInputValue] = useState('');
+    const [isError, setIsError] = useState(false);
```

Comment:
```suggestion
    const [isError, setError] = useState(false);
```

in boolean state it is ok to omit the prefix `is` and `should` it setState function 😉